diff --git a/dist/client/client-manager.js b/dist/client/client-manager.js
index 9ab2e7d57fd31781a7b802e3ac9054beb196e6f7..2692a507650b530ecd5e1eb336d1881bb61d220c 100644
--- a/dist/client/client-manager.js
+++ b/dist/client/client-manager.js
@@ -69,6 +69,7 @@ class ClientManager {
                     ? // Only expose 500 message if it was generated on another server
                         `Failed to fetch client information: ${err.message}`
                     : `Failed to fetch client information due to an internal error`;
+                    console.error("fetch error",err)
                 throw new invalid_client_metadata_error_js_1.InvalidClientMetadataError(message, err);
             }
             if (err instanceof zod_1.ZodError) {
